node/forest_libp2p/src/service.rs

```diff
@@ -364,8 +366,8 @@ pub fn build_transport(local_key: Keypair) -> Boxed<(PeerId, StreamMuxerBox), Er
         .into_authentic(&local_key)
         .expect("Noise key generation failed");
     let mut yamux_config = yamux::Config::default();
-    yamux_config.set_max_buffer_size(1 << 20);
-    yamux_config.set_receive_window(1 << 20);
+    yamux_config.set_max_buffer_size(16 * 1024 * 1024);
+    yamux_config.set_receive_window(16 * 1024 * 1024);
```

Comment:
Can you explain a little how these parameters are decided on? You said in the PR description the window size is now large enough, but 1 << 20 is larger than 16 * 1024 * 1024...

Reply:
nah it isn't (16_777_216 > 1_048_576) I updated the window size to be equal to the default go one, just to make sure no issues. 1<<20 was an arbitrary number I didn't think we would hit lol

Reply:
oh shoot, youre right.... *throws out my math degree*

Reply:
It looks smaller though, right?